test/e2e/apps/deployment.go

```diff
@@ -356,15 +356,16 @@ var _ = SIGDescribe("Deployment", func() {
 			if deployment, ok := event.Object.(*appsv1.Deployment); ok {
 				found := deployment.ObjectMeta.Name == testDeployment.Name &&
 					deployment.ObjectMeta.Labels["test-deployment-static"] == "true" &&
-					deployment.Status.ReadyReplicas == testDeploymentDefaultReplicas
+					deployment.Status.ReadyReplicas == testDeploymentMinimumReplicas
```

Comment:
This can't change. A few lines before we went from 1 to 2 replicas and this test ensure we reach that, so we can't change that suddenly to 1.